package.json

```diff
@@ -47,10 +47,10 @@
     "@tinyhttp/content-disposition": "^2.1.0",
     "html2canvas": "^1.4.1",
     "jspdf": "^2.5.1",
-    "jsts": "^2.9.3",
+    "jsts": "^2.10.0",
     "luxon": "^3.3.0",
     "nanoid": "^4.0.2",
-    "ol": "^7.4.0",
+    "ol": "^7.5.0",
```

Comment:
Probably we need to wait for https://github.com/openlayers/openlayers/issues/15004 to be fixed, see also https://github.com/openlayers/openlayers/pull/15005